core/templates/pages/blog-admin-page/blog-admin-page.mainpage.html

```diff
@@ -1,18 +0,0 @@
-<!DOCTYPE html>
-<html ng-app="oppia" lang="en">
-  <head>
-    <title>Oppia Blog Admin Panel</title>
-    <oppia-blog-admin-navbar></oppia-blog-admin-navbar>
```

Comment:
Do we need to move this to root.component?

Reply:
Yes, we need the navbar, apologies for overlooking it

Reply:
No problem! :) Can you please reply "Done" once this comment is resolved? (In general, reply to all threads as [suggested here](https://github.com/oppia/oppia/wiki/Make-a-pull-request#step-5-address-review-comments-until-all-reviewers-approve:~:text=Each%20reply%20should%20be%20either%20%22Done%22%20or%20a%20response%20explaining%20why%20the%20corresponding%20suggestion%20wasn%27t%20implemented.))

Reply:
Sure thing, this one's done!